src/components/v2/Layout/Header/CoinInfo.tsx

```diff
@@ -0,0 +1,34 @@
+/** @jsxImportSource @emotion/react */
+import React from 'react';
+import commaNumber from 'comma-number';
+import Typography from '@mui/material/Typography';
+import { Paper } from '@mui/material';
+import { useMarketsUser } from 'hooks/useMarketsUser';
+import { useVaiUser } from 'hooks/useVaiUser';
+import { Icon } from '../../Icon';
+import { useStyles } from './styles';
+
+const format = commaNumber.bindWith(',', '.');
+
+export const CoinInfo = ({ balance, coin }: { balance: string; coin: 'xvs' | 'vai' }) => {
+  const styles = useStyles();
+
+  return (
+    <Paper css={styles.coinInfo}>
+      <Icon name={coin} size="20px" />
+      <Typography variant="small1" noWrap component="p">
```

Comment:
Semantically speaking, this isn't a paragraph:
```suggestion
      <Typography variant="small1" noWrap component="span">
```